### ISSUE TITLE
First video missing on youtube home page when signed in [iPad]

### ISSUE THREAD

**Issue by Uni-verse** [brave/brave-ios#6666]
### Description: 
First video not shown on youtube home page when logged in

related to https://github.com/brave/brave-browser/issues/11955

### Steps to Reproduce 
  1. Install 1.46.x
  2. sign into google.com
  3. open youtube.com

**Actual result:** 
![IMG_0077](https://user-images.githubusercontent.com/17885425/209251968-5ab0b99f-bf5d-4f73-b9df-60ab9206f548.PNG)

**Expected result:**
![IMG_0078 2](https://user-images.githubusercontent.com/17885425/209252067-ec58922e-f9bb-4f50-84ba-0e8324690bf8.PNG)

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Easily

**Brave Version:** 
1.46 (22.12.15.19)

**Device details:** 
iPad

**Website problems only:**
- did you check with Brave Shields down?
- did you check in Safari/Firefox (WkWebView-based browsers)?

**Comment by Uni-verse** [comment]
Looks like its happening on desktop client as well. 

cc: @kjozwiak

**Comment by kjozwiak** [comment]
> Looks like its happening on desktop client as well.
> 
> cc: @kjozwiak

yup, also happening on `desktop` as well. I believe it's due to removing ad videos that usually appear in that spot. Maybe we can shift the videos over by one so we're not displaying this white space. Pretty sure this is due to cosmetic filtering. We had some issues in the past as some area's stay blank that have been blocked by Brave.

![image](https://user-images.githubusercontent.com/2602313/211072576-af74e499-9ac0-471d-8d03-f0d4ff0d7230.png)

**Comment by iccub** [comment]
Isn't this an ad taking first spot here?

**Comment by kjozwiak** [comment]
> Isn't this an ad taking first spot here?

@iccub yup, I guess the issue is to see if we can shift things over? Probably easier said then done but not sure if it's even worth it. I guess we can leave this opened as there's some users complaining about there being a blank area. But it's due to the ad video being removed as you mentioned above. I wouldn't consider this high priority though.

**Comment by iccub** [comment]
any opinion on that @ryanbr?

**Comment by ryanbr** [comment]
The blank space is just the uncollapsed advert that was removed. When we get procedural filters support in https://github.com/brave/brave-browser/issues/16935  it should help. I think we can all agree, having a blank space isn't ideal is far better than showing the advert.

`youtube.com##ytd-display-ad-renderer:upward(ytd-rich-item-renderer)`  The filter used in uBO. upward isn't currently supported

**Comment by iccub** [comment]
closing as wontfix for now

**Comment by ryanbr** [comment]
If https://github.com/brave/brave-browser/issues/27874 is supported on ios, this empry space should be fixed also
